sygnal/apnspushkin.py

```diff
@@ -422,25 +422,40 @@ def _get_payload_full(self, n, log):
                 badge = 0
             badge += n.counts.missed_calls
 
-        if badge is not None:
-            aps["badge"] = badge
-
-        if loc_key:
-            aps["mutable-content"] = 1
-
         if loc_key is None and badge is None:
             log.info("Nothing to do for alert of type %s", n.type)
             return None
 
         payload = {}
 
+        if (
+            n.type is not None
+            and device.data is not None
+            and device.data["default_payload"] is not None
+        ):
+            payload = device.data["default_payload"]
```

Comment:
same again, but because we now have multiple levels of dict, we'll have to do a proper [deepcopy](https://docs.python.org/3/library/copy.html#copy.deepcopy):

```suggestion
        if n.type and device.data:
            payload = copy.deepcopy(device.data.get("default_payload",{}))
```